manager/apis/app/v1alpha1/blueprint_types.go

```diff
@@ -211,6 +211,7 @@ func CreateMetaBlueprintWithoutState(blueprint *Blueprint) MetaBlueprint {
 }
 
 const (
-	BlueprintNamespaceLabel = "app.fybrik.io/blueprintNamespace"
-	BlueprintNameLabel      = "app.fybrik.io/blueprintName"
+	BlueprintNamespaceLabel        = "app.fybrik.io/blueprintNamespace"
+	BlueprintNameLabel             = "app.fybrik.io/blueprintName"
+	DataAccessModuleNamespaceLabel = "app.fybrik.io/dataAccessModuleNamespace"
```

Comment:
AFAIK usually dashed-case is used so I suggest to use `app.fybrik.io/modules-namespace` here and in a future PR do the same for all of our labels.

Reply:
If we change this label's name we also need to coordinate with the data control plane developers as they are now using this label; and any developers that have fybrikapplication yamls using this value. I would vote to leave it.

Reply:
I don't see why it makes any impact on existing users. This is an unreleased feature, if you set app.fybrik.io/dataAccessModuleNamespace and not the new label then the existing behavior is preserved: module will be installed to default namespace and the label will still be passed as-is to the module.